doc/sphinx/source/quickstart/install.rst

```diff
@@ -8,46 +8,63 @@ Paparazzi runs best on **Ubuntu 16.04 or higher**, so this quick installation gu
 
 Open a terminal and execute each lines below. If one fails, ask for help on gitter.
 
-Add paparazzi repositories and install dependencies:
+Version specific prerequisites
+------------------------------
+
+**If you have Ubuntu 20.04:**
+
+.. code-block:: bash
+
+    sudo apt-get install python-is-python3 gdb-multiarch
+
+**If you have Ubuntu 18.04 or lower:**
+
+.. code-block:: bash
+
+    sudo add-apt-repository -y ppa:team-gcc-arm-embedded/ppa
+    sudo apt-get update
+
+Install Paparazzi
+-----------------
+
+Add paparazzi apt-repository and install dependencies:
 
 .. code-block:: bash
 
-	sudo add-apt-repository -y ppa:paparazzi-uav/ppa
-	sudo add-apt-repository -y ppa:team-gcc-arm-embedded/ppa
-	sudo apt-get update
-	sudo apt-get -f -y install paparazzi-dev paparazzi-jsbsim gcc-arm-embedded
+    sudo add-apt-repository -y ppa:paparazzi-uav/ppa
+    sudo apt-get update
+    sudo apt-get -f -y install paparazzi-dev paparazzi-jsbsim gcc-arm-embedded
```

Comment:
`gcc-arm-embedded` is the name of the package from ppa, but when installed from ubuntu it is `gcc-arm-none-eabi`, so I think it should be in the specific part.